google/cloud/forseti/services/client.py

```diff
@@ -797,3 +799,17 @@ def delete_model(self, model_name):
         """
 
         return self.model.delete_model(model_name)
+
+    @property
+    def interceptors(self):
```

Comment:
Why is this a property method here in the `services/client.py`, but `setup_interceptors()` in `services/server.py`?

Reply:
Got it, thanks.